src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -1145,6 +1153,7 @@ namespace Akka.Actor
         public Akka.Actor.IActorRef DeadLetters { get; }
         public Akka.Actor.Address DefaultAddress { get; }
         public Akka.Actor.Deployer Deployer { get; }
+        public Akka.Event.EventStream EventStream { get; }
```

Comment:
Added `EventStream` to `LocalActorRefProvider` as a public property, so the `EventStream` could be accessed by parts of the `RemoteActorRefProvider`